packages/babel-preset-default/package.json

```diff
@@ -22,12 +22,14 @@
 		"node": ">=8"
 	},
 	"dependencies": {
+		"@babel/plugin-proposal-async-generator-functions": "^7.0.0-beta.49",
+		"@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.49",
+		"@babel/plugin-transform-react-jsx": "^7.0.0-beta.49",
+		"@babel/plugin-transform-runtime": "^7.0.0-beta.49",
+		"@babel/preset-env": "^7.0.0-beta.49",
+		"@babel/runtime": "^7.0.0-beta.49",
```

Comment:
I'm wondering if we shouldn't apply `runtime` to all packages that have it referenced after Babel does its magic.

Reply:
I'm wondering if we can't just remove the runtime since we're using the polyfills of `babel-env`

Reply:
Hmmm, `@babel/runtime` and `core-js` are used for requiring code in the transpiled code when using `env` preset. The latter is the dependency of the previous one.